server/settings/publish_plugins.py

```diff
@@ -70,53 +70,65 @@ class PublishPluginsModel(BaseSettingsModel):
     ValidateAbsoluteDataBlockPaths: ValidatePluginModel = SettingsField(
         default_factory=ValidatePluginModel,
         title="Validate Absolute Data Block Paths",
+        description="Checks if external data having absolute file path.",
         section="General Validators"
     )
     ValidateCameraZeroKeyframe: ValidatePluginModel = SettingsField(
         default_factory=ValidatePluginModel,
-        title="Validate Camera Zero Keyframe"
+        title="Validate Camera Zero Keyframe",
+        description="Checks if keyframe set on the zero frame - mostly when asset publish to Unreal Engine.",
     )
     ValidateFileSaved: ValidateFileSavedModel = SettingsField(
         default_factory=ValidateFileSavedModel,
         title="Validate File Saved",
+        description="Checks if workfile changes been saved, forcing user to perform save action if not.",
     )
     ValidateInstanceEmpty: ValidatePluginModel = SettingsField(
         default_factory=ValidatePluginModel,
-        title="Validate Instance is not Empty"
+        title="Validate Instance is not Empty",
+        description="Checks for empty publish instances which could result into no data when published.",
     )
     ValidateMeshHasUvs: ValidatePluginModel = SettingsField(
         default_factory=ValidatePluginModel,
         title="Validate Mesh Has Uvs",
+        description="Checks for geometry having UV set defined.",
         section="Model Validators"
     )
     ValidateMeshNoNegativeScale: ValidatePluginModel = SettingsField(
         default_factory=ValidatePluginModel,
-        title="Validate Mesh No Negative Scale"
+        title="Validate Mesh No Negative Scale",
+        description="Checks for object scaling in negative values which is prohibited then.",
     )
     ValidateModelMeshUvMap1: ValidatePluginModel = SettingsField(
         default_factory=ValidatePluginModel,
-        title="Validate Model Mesh Has UV map named map1"
+        title="Validate Model Mesh Has UV map named map1",
+        description="Checks if objects have a UV set named 'map1' meeting Maya standards.",
     )
     ValidateTransformZero: ValidatePluginModel = SettingsField(
         default_factory=ValidatePluginModel,
-        title="Validate Transform Zero"
+        title="Validate Transform Zero",
+        description="Checks for zero values in the objects transforms.",
     )
     ValidateNoColonsInName: ValidatePluginModel = SettingsField(
         default_factory=ValidatePluginModel,
-        title="Validate No Colons In Name"
+        title="Validate No Colons In Name",
+        description="Checks for invalid naming format containing : within name - mostly when exporting to Unreal Engine.",
```

Comment:
```suggestion
        description="Checks for invalid naming format containing `:` within name - mostly when exporting to Unreal Engine.",
```

I believe the tooltips support markdown.